server/fixity.go

```diff
@@ -215,6 +222,39 @@ func (s *RESTServer) GetFixityIdHandler(w http.ResponseWriter, r *http.Request,
 	enc.Encode(thisFixity)
 }
 
+// DeleteFixity handles requests to DELETE /fixty/:id
+// This deletes a scheduled fixity check for the given id . reurns 404 ff no request is found in the 'scheduled' state for the give id, 
+// 200 otherwise.
+func (s *RESTServer) DeleteFixityHandler(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+        id := ps.ByName("id")
+        err := s.FixityDatabase.DeleteFixity(id)
+        if err != nil {
+                w.WriteHeader(500)
+                fmt.Fprintln(w, err.Error())
+        }
+}
+
+func (s *RESTServer) PutFixityHandler(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	id := ps.ByName("id")
+	return_code, err := s.FixityDatabase.PutFixity(id)
```

Comment:
nope nope nope nope nope.

I dislike the database code returning an http response. because now we need to keep both database adapters synced, and also, it moves the http specific logic outside of the handler.

Please rework.

Reply:
Done.